roles/nova/vars/main.yml

```diff
@@ -107,6 +107,7 @@ _nova_helm_values:
         ovsdb_connection: unix:/run/openvswitch/db.sock
       scheduler:
         workers: 8
+        discover_hosts_in_cells_interval: 300
```

Comment:
Let's switch this to 30, it makes more sense in our environments and it will appear faster.

Reply:
OK